src/LeanCode.ContractsGenerator/Program.cs

```diff
@@ -11,14 +11,18 @@ public interface IOptions
 {
     public const string StdoutMarker = "-";
 
-    [Option('o', "output", Required = true, MetaValue = "FILE", HelpText = "The output file path. Use `-` to write the resulting Protobuf file to `stdout`.")]
+    [Option('o', "output", MetaValue = "FILE", HelpText = "The output file path. Use `-` to write the resulting Protobuf file to `stdout`.", Group = "output")]
     public string OutputFile { get; set; }
+
+    [Option("check-only", HelpText = "Check the contracts only. Do not generate output.", Group = "output")]
+    public bool CheckOnly { get; set; }
```

Comment:
Isn't it kind of redundant? The same can already be achieved by providing `--output /dev/null` and there are no other side effects that we would need to suppress anyway.

Reply:
DX of such a solution is... Non-existent. ;) Also, this assumes you work on *nix systems.

Reply:
Well, yes, there are *nix systems and there is Windows on which one could probably use `--output nul`. Do we support anything else? :upside_down_face:

And about discoverability, isn't it actually very common for tools that process some files and produce one file of output to not bother with such switches? `gcc`, `csc`, `dart compile exe` don't seem to at least. `tsc` does but it can produce multiple files.

Reply:
Which would require special case either way. ;)

It makes sense for tools like these to _not_ have this kind of switch, because not producing output does not give you value (you have to have the result either way).

Here, we want to have dry runs as a first class citizen. Developer experience of such a solution is much, much higher than piping the output to `/dev/null`.